src/network/networkd-network.c

```diff
@@ -481,7 +481,13 @@ int network_load_one(Manager *manager, OrderedHashmap **networks, const char *fi
                               "IPv6Prefix\0"
                               "IPv6RoutePrefix\0"
                               "TrafficControlQueueingDiscipline\0"
-                              "CAN\0",
+                              "CAN\0"
+                              "ControlledDelay\0"
+                              "FairQueueTrafficPolicing\0"
```

Comment:
this is known as FQ so why not just "FairQueue"?

Reply:
Good point too.